.cursorrules

```diff
@@ -0,0 +1,114 @@
+# Phel IntelliJ Plugin Development Rules
+
+## Project Context
+
+This is an IntelliJ IDEA plugin for the [Phel programming language](https://phel-lang.org/) - a functional programming language that compiles to PHP. Phel is a Lisp dialect inspired by Clojure and Janet.
+
+## Official Documentation References
+
+- Official JetBrains Plugin SDK: https://plugins.jetbrains.com/docs/intellij/welcome.html
+- GitHub Documentation root: https://github.com/phel-lang/phel-lang.org/tree/master/content/documentation
+
+## Language Specification Compliance
+
+Always refer to the official Phel documentation as the single source of truth. Key language features:
+
+### Comments
+
+- Line comments start with `#` character
+- No block comments supported
+
+### Keywords
+
+- Simple keywords: `:keyword`, `:keys`, `:simple-name`
+- Namespaced keywords: `:namespace/keyword`, `:php/array`
+- Current namespace shortcuts: `::foo` (evaluates to `:current-namespace/foo`)
+- Aliased namespace shortcuts: `::alias/foo` (using namespace aliases)
+- Mixed separators: `:my\namespace/keyword` (backslash + forward slash)
+
+### PHP Interoperability
+
+- Static method calls: `(php/:: "ClassName" "methodName")`
+- Instance method calls: `(php/-> $object "method")`
```

Comment:
@JesusValeraDev objects without $ in phel. See https://phel-lang.org/documentation/php-interop/#php-method-and-property-call